doc/iterations/2024.9/add_support_for_diagram_tool_shortcuts.adoc

```diff
@@ -0,0 +1,69 @@
+= Add support for diagram tool shortcuts
+
+== Problem
+
+Diagram tools are accessible only from the contextual palette of the diagram, a node or an edge
+
+== Key Result
+
+- Specifiers is able to define the shortcut associated to a node or an edge tool in the View model.
```

Comment:
Edge tools were explicitly excluded since they bring additional issues (what is the source, what is the target, etc)

Reply:
Done